scripts/integration/runPost.py

```diff
@@ -19,6 +20,11 @@ def runPost(inPath, outPath, conos):
     else:
         adata = scib.pp.read_seurat(inPath)
 
+    # quick fix `TypeError: Object dtype dtype('O') has no native HDF5 equivalent` when saving
+    obs_key = 'nFeature_RNA'
+    if obs_key in adata.obs:
+        adata.obs[obs_key] = adata.obs[obs_key].astype(np.int32)
```

Comment:
could you generally implement this for other data using `adata.obs.convert_dtypes`? (https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.convert_dtypes.html)

Reply:
You mean something like this?
```py
adata.obs = adata.obs.convert_dtypes()
```

Reply:
Haven't tested it, but I was thinking sth like that yes.

Reply:
Didn't work, unfortunately, as the problem appears when pandas dtype classes are being written instead of the older dtypes. I solved the issue instead by pinning `anndata>=0.8`. Only reason for lower versions was mnnpy, which we are currently ignoring anyway